### PR TITLE
feat(docs): add support and community documentation

### PR DESCRIPTION
**Pull request by mason-at-pieces**
Changes:
- [x] Changed the `/our-community` page to be `/community` page
- [x] Replaced `YouTube`, `Discord`, `Open Source`, and `Contact Support` links from the sidebar with `Community` and `Support` links
  - Will make sure that the replaced links are either on the `/community` or `/support` page
- [x] Removed `Our Community` from the `Community & More` section
- [x] Renamed `Community & More` section to `FAQs & More`
- [x] Updated sitemap link from `/our-community` to `/community`
- [x] Linked `/support` in the sidebar
- [x] Added `/support` page with copy
- [x] Finalize copy

### CODE REVIEW

**Comment by mason-at-pieces** [comment]
Work done:
- Changed the `our-community` page to be `community` page
- Replaced `YouTube`, `Discord`, `Open Source`, and `Contact Support` links from the sidebar with `Community` and `Support` links
- Removed `Our Community` from the `Community & More` section
- Renamed `Community & More` section to `FAQs & More`
- Updated sitemap link from `our-community` to `community`

**Comment by jordan-pieces** [comment]
@mason-at-pieces just wanted to check on the status here, mark as ready for review whenever you are ready

File: docs/sitemap.xml [hunk 1]
```diff
@@ -133,7 +133,7 @@
   <priority>0.80</priority>
 </url>
 <url>
-  <loc>https://docs.pieces.app/our-community</loc>
+  <loc>https://docs.pieces.app/community</loc>
```
Comment (jordan-pieces) [inline docs/sitemap.xml:136]: beauty

**Review by jordan-pieces — APPROVED** [review]
@mason-at-pieces looking great, if there arent any other changes we can get this up!

**Comment by mason-at-pieces** [comment]
@jordan-pieces This is good to ship 👍

**Pull request merged**